x-pack/qa/sql/security/ssl/build.gradle

```diff
@@ -207,8 +206,16 @@ integTestCluster {
   }
 }
 
-
-
+Closure notRunningFips = {
+  Boolean.parseBoolean(BuildPlugin.runJavascript(project, project.runtimeJavaHome,
+          'print(java.security.Security.getProviders()[0].name.toLowerCase().contains("fips"));')) == false
+}
+// Do not attempt to form a cluster in a FIPS JVM, as doing so with a JKS keystore will fail.
+// TODO Revisit this when SQL CLI client can handle key/certificate instead of only Keystores.
```

Comment:
is there an issue open for this?

Reply:
No, we discussed it with Costin but I didn't yet come around to opening the issue. Will do ASAP

Reply:
thanks and please add it to the code comment